python/python/ert/job_queue/queue.py

```diff
@@ -187,20 +187,26 @@ def num_running( self ):
         return self._num_running( )
 
     def num_pending( self ):
-        return self._.num_pending( )
+        return self._num_pending( )
 
     def num_waiting( self ):
         return self._num_waiting( )
 
     def num_complete( self ):
         return self._num_complete( )
 
+    def __getitem__(self, index):
+        idx = index
+        ls = len(self)
+        if idx < 0:
+            idx += ls
+        if 0 <= idx < ls:
+            return self._iget_driver_data(idx)
+        raise IndexError('index out of range, was: %d should be in [0, %d)' %
+                         (index, ls))
+
     def exists(self, index):
-        job = self.__getitem__(index)
-        if job:
-            return True
-        else:
-            return False
+        return self[index]
```

Comment:
To be fair, I don't know what the intended usage was; the `__getitem__` wasn't implemented prior to this patch!

Please submit an issue if `exists` should be used for something else than this.